config.json

```diff
@@ -53,6 +53,19 @@
       ]
     },
     {
+      "core": true,
+      "difficulty": 2,
+      "slug": "nucleotide-count",
+      "topics": [
+        "filtering",
+        "pattern-matching",
+        "strings",
+        "test-driven-development"
+      ],
+      "unlocked_by": "rna-transcription",
+      "uuid": "9debee8e-003d-4eb2-bf97-6d3764d024ca"
+    },
+    {
```

Comment:
I put it here, because it is related to the rna-transcriptor. That's why I let it unlock with that exercise.
The uuid I generated with https://www.uuidgenerator.net